doc/source/serve/api/index.md

```diff
@@ -44,31 +44,44 @@ This is fixed by added custom filename mappings in `source/conf.py` (look for "a
    serve.handle.RayServeSyncHandle
 ```
 
-#### Advanced APIs
+### Running Applications
 
 ```{eval-rst}
 .. autosummary::
    :nosignatures:
    :toctree: doc/
 
-   serve.get_replica_context
-   serve.get_multiplexed_model_id
-   serve.get_app_handle
-   serve.get_deployment_handle
+   serve.run
+   serve.delete
+   serve.start
+   serve.shutdown
+   serve.status
 ```
 
-### Running Applications
+### Configuring System-level Options
 
 ```{eval-rst}
 .. autosummary::
    :nosignatures:
    :toctree: doc/
 
-   serve.run
-   serve.delete
    serve.start
-   serve.shutdown
-   serve.status
+   serve.config.ProxyLocation
+   serve.config.HTTPOptions
+```
+
+#### Advanced APIs
+
+```{eval-rst}
+.. autosummary::
+   :nosignatures:
+   :toctree: doc/
+
+   serve.get_replica_context
+   serve.context.ReplicaContext
+   serve.get_multiplexed_model_id
+   serve.get_app_handle
+   serve.get_deployment_handle
```

Comment:
Will you be adding the new `DeploymentHandle` to api refs in a separate PR?
<img width="761" alt="image" src="https://github.com/ray-project/ray/assets/15851518/5851d3a6-bf1b-4da8-91d3-838a38d7e17d">

Reply:
yep, will add them separately